packages/cli-repl/src/mongosh-repl.ts

```diff
@@ -485,6 +485,7 @@ class MongoshNodeRepl implements EvaluationListener {
    *
    * @param _initializationToken A value obtained by calling {@link MongoshNodeRepl.initialize}.
    */
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   async startRepl(_initializationToken: InitializationToken): Promise<void> {
```

Comment:
Just curious: if this is not used, do we need it here?

Reply:
Don't know. Maybe a typecheck? Can it be removed @addaleax ?

Reply:
It’s for typechecking, yes, see the comment for the type definition:

https://github.com/mongodb-js/mongosh/blob/272e94190fdf16cc71cdb86be88657a2a02a26c2/packages/cli-repl/src/mongosh-repl.ts#L85-L88